src/main/webapp/src/components/songInfo.js

```diff
@@ -0,0 +1,208 @@
+        "Now the workers have struck for fame\n" +
+        "'Cause Lennon's on sale again\n" +
+        "See the mice in their million hordes\n" +
+        "From Ibiza to the Norfolk Broads\n" +
+        "Rule Britannia is out of bounds\n" +
+        "To my mother, my dog, and clowns\n" +
+        "But the film is a saddening bore\n" +
+        "'Cause I wrote it ten times or more\n" +
+        "It's about to be writ again\n" +
+        "As I ask you to focus on\n" +
+        "Sailors fighting in the dance hall\n" +
+        "Oh man, look at those cavemen go\n" +
+        "It's the freakiest show\n" +
+        "Take a look at the lawman\n" +
+        "Beating up the wrong guy\n" +
+        "Oh man, wonder if he'll ever know\n" +
+        "He's in the best selling show\n" +
+        "Is there life on Mars?",
+      sentimentAnalysis: {
+        score: "-0.6",
+        magnitude: "1.3",
+      },
+      entityAnalysis: [
+        {
+          word: "affair",
+          salience: "0.51",
+          category: "work of art",
+        },
+        {
+          word: "girl",
+          salience: "0.09",
+          category: "person",
+        },
+        {
+          word: "film",
+          salience: "0.06",
+          category: "work of art",
+        },
+        {
+          word: "selling show",
+          salience: "0.04",
+          category: "work of art",
+        },
+        {
+          word: "God",
+          salience: "0.01",
+          category: "person",
+        },
+        {
+          word: "Mars",
+          salience: "0.01",
+          category: "location",
+        },
+      ],
+      youTubeRecommendations: [
+        "AZKcl4-tcuo",
+        "2zX0lxiM8Xo",
+        "kJ095S0MmnA",
+        "slIG7Aes7DM",
+        "uqeH76RLpO8",
+      ],
+    };
+
+    // TODO display entities with charts
+    return (
+      <div className={classes.root}>
+        <Typography variant="h3" className={classes.songName}>
+          "{songInfo.songName}"
+        </Typography>
+        <div className={classes.languageAnalysisSection}>
+          <div className={classes.songLyrics}>
+            <Typography variant="h4">Lyrics</Typography>
+            <p>{songInfo.lyrics}</p>
+          </div>
+          <div>
+            <div class="song-sentiment-analysis">
+              <Typography variant="h4">Sentiment Analysis</Typography>
+              <p>Score: {songInfo.sentimentAnalysis.score}</p>
+              <p>Magnitude: {songInfo.sentimentAnalysis.magnitude}</p>
+            </div>
+            <div class="song-entity-analysis">
+              <Typography variant="h4">Entity Analysis</Typography>
+              <List>
+                {songInfo.entityAnalysis.map((wordInfo, index) => (
+                  <ListItem key={index + 1}>
+                    {wordInfo.word +
+                      " - " +
+                      wordInfo.salience +
+                      " - " +
+                      wordInfo.category}
```

Comment:
nit: probably it's better to use something like  [react-string-format)](https://www.npmjs.com/package/react-string-format) for that. It would look like `format('{0} - {1} - {2}', wordInfo.word, wordInfo.salience, wordInfo.category)`

Reply:
sure, it was made just to display info, because it'll change vey soon by charts